app/views/provider_interface/interviews/_form.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= f.govuk_date_field :date %>
+<%= f.govuk_date_field :date, hint: { text: t('helpers.hint.provider_interface_interview_wizard.date', example_date: @wizard.example_date) } %>
```

Comment:
This form should really be a component and then we can test that it's rendering sensible things. Looking into that now